packages/notification/src/types.tsx

```diff
@@ -24,5 +24,5 @@ export interface KRTNotificationNativeModule {
 }
 
 export interface RemoteMessage {
-  data?: { [key: string]: string };
+  data?: { [key: string]: string | object };
```

Comment:
This change is a diff of this PR
https://github.com/plaidev/karte-react-native/pull/58